From 19bcd03eec68888203df35a1d9510a2ac15e68b3 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 4 Feb 2016 15:35:01 +0000
Subject: [PATCH 199/462] MANTIS 0040084 extension: Admin-Application: allow
 client to send notification to all clients

added JarApplicationAdapter for generated files from ApplicationAdapter.xsd for AdminApplication


git-svn-id: svn://localhost/SelfServiceBuild/trunk@303 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index bbf8cce..bb81539 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -239,6 +239,7 @@
 <property name="JarMassaiStatechart"        value="${JarDir}/massai-statechart.jar"/>
 <property name="JarMatacClient"             value="${JarDir}/MATAC-CLIENT.jar"/>
 <property name="JarMatac"                   value="${JarDir}/MATAC-XBEANS.jar"/>
+<property name="JarApplicationAdapter"      value="${JarDir}/ApplicationAdapter.jar"/>
 
 
 <!-- Definition des IDL-Compilers: OpenORB ======================= -->
-- 
2.41.0.windows.1

